pkgs/by-name/co/construct/package.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, boost
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "construct";
+  version = "master";
```

Comment:
here is a guide to what version should be

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming